src/haptic_feedback.h

```diff
@@ -27,7 +27,8 @@ typedef enum {
     HAPTIC_FEEDBACK_DUTY,
     HAPTIC_FEEDBACK_DUTY_CONTINUOUS,
     HAPTIC_FEEDBACK_ERROR_TEMPERATURE,
-    HAPTIC_FEEDBACK_ERROR_VOLTAGE,
+    HAPTIC_FEEDBACK_ERROR_LO_VOLTAGE,
+    HAPTIC_FEEDBACK_ERROR_HI_VOLTAGE,
```

Comment:
I don't think the one or two extra chars will change anything, so I'd prefer the `HIGH` and `LOW` spelling.